docs/src/api/class-playwright.md

```diff
@@ -83,6 +83,57 @@ class PlaywrightExample
 }
 ```
 
+## async method: Playwright._newRequest
+* langs: js
+- returns: <[FetchRequest]>
+
+**experimental** Creates new instances of [FetchRequest].
+
+### option: Playwright._newRequest.useragent
```

Comment:
You can use references here to avoid copying the docs. See `docs/src/test-api/class-testoptions.md:126` for example.

Reply:
So far they are only used in one place, I'll extract them into parmams.md if we decide to pass them to individual requests.